canvas_modules/harness/cypress/support/canvas/verification-cmds.js

```diff
@@ -1369,9 +1369,21 @@ Cypress.Commands.add("verifyNotificationIconType", (type) => {
 });
 
 Cypress.Commands.add("verifyCanvasTransform", (movString) => {
-	cy.get("#canvas-div-0 .d3-canvas-group")
-		.invoke("attr", "transform")
-		.should("eq", movString);
+	// Verify the argument passed is a string
+	if (movString === String) {
```

Comment:
This isn't how you test for a string. This line of code is the reason invalid test situations are returning success (as mentioned in my comment above). You should use `typeof` to test for the type of a variable.